### PR TITLE
Add: support TLS in the helm chart.

### PR DESCRIPTION
**Pull request by jjnicola**
**What**:
Add tls certificate configuration to the helm charts. Now the service listen into https:// instead of http://

Jira: SC-918

**Why**:

**How**:
Set the environment variables and run the smoke test. For more information, see https://github.com/greenbone/openvas-scanner/tree/main/rust/smoketest#readme

**Checklist**:

- [ ] Tests
- [ ] PR merge commit message adjusted

### CODE REVIEW

File: rust/smoketest/README.md [hunk 1]
```diff
@@ -36,14 +38,13 @@ All settings for running the smoke-tests are set via environmental variables. Th
 # set env variables
 export CLIENT_CERT=/tmp/cert.pem
 export CLIENT_KEY=/tmp/key.pem
-export OPENVASD=192.168.0.1:3000
```
Comment (Kraemii) [inline rust/smoketest/README.md:39]: In the table above is still `OPENVASD`

File: rust/smoketest/README.md [hunk 2]
```diff
@@ -36,14 +38,13 @@ All settings for running the smoke-tests are set via environmental variables. Th
...
+export OPENVASD_SERVER=192.168.0.1:3000
 export TARGET_HOSTNAME=192.168.10.10
 export TARGET_USERNAME=user
 export TARGET_PASSWORD=pass
 export API_KEY=mtls_is_preferred
 
 #build and run
-make build 
+make build
 make run-with-certs
```
Comment (Kraemii) [inline rust/smoketest/README.md:49]:
```suggestion
make run
```

File: rust/smoketest/README.md (hunk 1 above)
Comment (jjnicola) [reply rust/smoketest/README.md:39]: I have updated the var name in the table

File: rust/smoketest/README.md [hunk 3]
```diff
@@ -36,14 +37,13 @@ All settings for running the smoke-tests are set via environmental variables. Th
 # set env variables
 export CLIENT_CERT=/tmp/cert.pem
 export CLIENT_KEY=/tmp/key.pem
-export OPENVASD=192.168.0.1:3000
+export OPENVASD_SERVER=192.168.0.1:3000
 export TARGET_HOSTNAME=192.168.10.10
 export TARGET_USERNAME=user
 export TARGET_PASSWORD=pass
 export API_KEY=mtls_is_preferred
```
Comment (Kraemii) [inline rust/smoketest/README.md:45]:
```suggestion
export API_KEY=mtls_is_preferred
export SCAN_CONFIG=...

```

File: rust/smoketest/README.md [hunk 4]
```diff
@@ -24,7 +25,7 @@ All settings for running the smoke-tests are set via environmental variables. Th
 |TARGET_USERNAME|Username for login in the target during the authenticated scan|empty string|no|Necessary for authenticated scans|
 |TARGET_PASSWORD|Password for login in the target during the authenticated scan|empty string|no|Necessary for authenticated scans|
 |API_KEY|API Key for authenticated communication with `openvasd`|mtls_is_preferred|yes||
```
Comment (Kraemii) [inline rust/smoketest/README.md:27]: Is the API key mandatory? In the `config.rs` it is optional, and also the default value is not set, so would be None

File: rust/smoketest/Makefile [hunk 5]
```diff
@@ -12,14 +12,17 @@ else
...
 
 ifndef SCAN_CONFIG
-SCAN_CONFIG := ./simple_scan_ssh_only.json
+SCAN_CONFIG := configs/simple_scan_ssh_only.json
+endif
+
+ifndef CLIENT_CERT
+CLIENT_CERT := configs/client_sample.cert
+CLIENT_KEY := configs/client_sample.key
 endif
```
Comment (Kraemii) [inline rust/smoketest/Makefile:26]: Should the certs be set by default? They are not needed, when mTLS is disabled

**Review by Kraemii — APPROVED** [review]

**Pull request merged**